src/docfx/lib/markdown/MarkdownEngine.cs

```diff
@@ -69,7 +69,7 @@ internal class MarkdownEngine
                     new ContentValidationContext(
                         fileResolver.ResolveFilePath(config.MarkdownValidationRules),
                         fileResolver.ResolveFilePath(config.Allowlists),
-                        fileResolver.ResolveFilePath(config.Disallowlists)),
+                        ""),
```

Comment:
disallow list are not used in ApexValdiation Lib, so I just set it to empty string, once we retire ApexValidation lib, it can be removed totally.